products/cfengine.md

```diff
@@ -0,0 +1,46 @@
+---
+title: CFEngine
+layout: post
+command: cf-agent --version
+chagnelogTemplate: https://github.com/cfengine/core/blob/__RELEASE_CYCLE__.x/ChangeLog
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+activeSupportColumn: true
+eolColumn: Supported
+iconSlug: NA
+permalink: /cfengine
+
+releases:
+  - releaseCycle: "3.18"
+    release: 2021-06-24
+    lts: true
+    latest: "3.18.0"
+    support: 2023-06-24
+    eol: false
+  - releaseCycle: "3.17"
+    release: 2020-11-18
+    lts: false
+    eol: true
+    support: false
+  - releaseCycle: "3.16"
+    release: 2020-06-25
+    lts: false
+    latest: "3.17.0"
```

Comment:
^^ this one should be above, you moved here by mistake ( 3.17.0 moved inside 3.16 here )
And you dont need to add lts: false tag if it is not lts

Reply:
These 2 are the last things from me , other than these 2 it is OK from my side

Reply:
Thanks, fixed.

Reply:
Thank you :)